cypress/e2e/attribute_validations/reporting.cy.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from failing the test
+  return false
+})
+
+describe('Add multiple clusters and search', () => {
+  // This will hold the fixture data and the flag for skipping the tests
+  let skipTest = false
+
+  before(function () {
+    // Load the fixture data before running any tests
+    cy.fixture('data').then((data) => {
+      this.data = data
+      // If mode is "matter", set skipTest flag to true
+      if (this.data.mode === 'matter') {
```

Comment:
Better to run it for zigbee than not matter. If a new protocol does not have attribute reporting then this test will fail.

Reply:
Sure good idea

Reply:
done, thank you